app/views/content/train-to-be-a-teacher/school-placements.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Teacher training school placements"
+heading: "Teacher training school placements"
+subcategory: Postgraduate teacher training
+description: |-
+  Learn about teacher training school placements, including how long they are, what trainees do on placements, and the support available. 
+navigation: 20.20
+navigation_title: Teacher training school placements
+navigation_description: Learn about teacher training school placements, including how long they are, what trainees will do on placements, and the support available. 
+quote:
+  q-giorgio:
+    text: "The highlight of my placements was getting to teach!"
+    name: "Giorgio, biology teacher"
+    classes: quote--indent
+promo_content:
+    - content/train-to-be-a-teacher/promos/adviser-promo-apply
+keywords:
+    - placements
+    - school placements
+    - classroom placements
+    
+---
+
+School placements are an important and exciting part of school-led or university-led postgraduate teacher training. They're a chance to:
+
+* get experience in the classroom
+* immerse yourself in your placement schools
+* get to know your pupils and colleagues
+* start your career as a teacher
+
+## How long are placements?
+
+School placements last for a minimum of 24 weeks. This time is spread across at least 2 schools. This is to give you a varied and thorough experience in the classroom. For example, in your first placement you might teach key stage 4, and in your second placement you might teach key stage 3 in a school for pupils with [special educational needs and disabilities (SEND)](/life-as-a-teacher/age-groups-and-specialisms/special-educational-needs). 
+
+## Where will my placements take place? 
+
+Your training provider will select placement schools for you. They will contact you and discuss your situation to help them select a location that you can travel to. 
+
+## What will I do on my placements?  
+
+You school placement aims to give you enough experience to: 
+
+* meet all the [teachers’ standards](https://www.gov.uk/government/publications/teachers-standards)
+
+* achieve [qualified teacher status (QTS)](/train-to-be-a-teacher/what-is-qts) 
+
+On your school placement, you'll get hands-on experience of what it's like to be a teacher. So, you’ll be spending a lot of time in the classroom learning how to teach lessons. Beyond the classroom, you can also get involved in other aspects of school life, like school trips, school plays, parents' evenings and staff meetings.  
+
+Over your placement you’ll get to see your pupils progress and get an insight into how rewarding teaching can be.  
+
+$q-giorgio$
+
+## What support will I get during my placements? 
+
+School placements can be daunting, but there's lots of support available.
+
+### Your skills from training
+You’ll start your placements well equipped with a toolkit of theory and techniques that you'll learn in training. Placements are a chance to put that into practice.  
```

Comment:
Should this be 'that you've learnt in training'?

Reply:
I don't think so, because this page should be targeting people who haven't started or applied to training yet? Unless we do 'you will have learnt'?